pages/ksphere/konvoy/1.5/concepts/index.md

```diff
@@ -13,7 +13,7 @@ The topics in this section provide a brief overview of the native Kubernetes arc
 
 The following diagram provides a simplified architectural overview to help you visualize the key components of the cluster:
 
-![Architectural overview](../img/Konvoy-arch-diagram.png)
+![Architectural overview](../img/Konvoy-arch-diagram.png){ data-no-border }
```

Comment:
@cmastr you can remove borders like this. 

it's important that there's no whitespace. 

```
# WORKS
![Architectural overview](../img/Konvoy-arch-diagram.png){ data-no-border }

# WON'T WORK
![Architectural overview](../img/Konvoy-arch-diagram.png)  { data-no-border }
```

Reply:
Ok good to know! I need to add this to the wiki as well.

Reply:
I'll add this to the readme too.

Reply:
done